dwave_networkx/drawing/qubit_layout.py

```diff
@@ -468,34 +526,237 @@ def draw_yield(G, layout, perfect_graph, unused_color=(0.9,0.9,0.9,1.0),
 
-    # Draw rest of graph
-    if unused_color is not None:
-        if nodelist is None:
-            nodelist = G.nodes() - faults_nodelist
-        if edgelist is None:
-            edgelist = G.edges() - faults_edgelist
+    Parameters
+    ----------
 
-        unused_node_color = [unused_color for v in nodelist]
-        unused_edge_color = [unused_color for v in edgelist]
+    G : networkx.Graph
+        A graph constructed by :func:`chimera_layout`, :func:`pegasus_layout, or
+        :func:`zephyr_layout`
+
+    pos : dict
+        A dictionary with nodes as keys and 2x2 matrices [[x0, y0], [x1, y1]]
+        representing the line segments of nodes.
+
+    ax : matplotlib.Axis
+        The matplotlib Axis object to draw the graph on.
+
+    node_size : float
+        The size (in area) of the circles used to depict internal edges.
+
+    width : float
+        The width of line segments associated with edges, and half the width of
+        line segments associated with nodes.
+
+    nodelist : iterable or None (default=None)
+        The set of nodes to draw.  If None, all nodes from G are drawn.
+
+    edgelist : iterable or None (default=None)
+        The set of edges to draw.  If both nodelist and edgelist are None, all
+        edges of G are drawn.  If edgelist is None, all edges from the subgraph
+        ``G.subgraph(nodelist)`` are drawn.
+
+    node_color : iterable or string (default='blue')
+        The sequence of colors to use in drawing nodes of G.  If node_color is
+        not a string, the colors are taken in the same order as nodelist, and
+        each color is either a float, a 3-tuple or 4-tuple of floats.
+
+    edge_color : iterable or string (default='black')
+        The sequence of colors to use in drawing edges of G.  If edge_color is
+        not a string, the colors are taken in the same order as edgelist, and
+        each color is either a float, a 3-tuple or 4-tuple of floats.
+
+    cmap : string or matplotlib.ColorMap or None (default=None)
+        A colormap to color nodes with.  Presumes that node_color is a sequence
+        of floats.
+
+    vmin : float or None (default=None)
+        Minimum value to use to use when normalizing node colors through cmap.
+
+    vmax : float or None (default=None)
+        Maximum value to use to use when normalizing node colors through cmap.
+
+    edge_cmap : string or matplotlib.ColorMap or None (default=None)
+        A colormap to color edges with.  Presumes that edge_color is a sequence
+        of floats.
+
+    edge_vmin : float or None (default=None)
+        Minimum value to use to use when normalizing edge colors through
+        edge_cmap
+
+    edge_vmax : float or None (default=None)
+        Maximum value to use to use when normalizing edge colors through
+        edge_cmap
+    
+    z_offset : int (default=0)
+        An offset to the zorder that various elements are drawn in.  Edge lines
```

Comment:
Perhaps "zorder" for first usage before following explanation

Reply:
The offset is absolute, and set per lineplot.  This functionality is used by `draw_qubit_embedding`, where we first draw the fabric of nodes and edges which are not used by the embedding and then draw nodes and edges used by the embedding -- the unused pieves have a lower `z_offset` so that they are drawn beneath the used portions.